### ISSUE TITLE
I want to be able to specify the default position of the setting on the screen

### ISSUE THREAD

**Issue by HL2222** [igrigorik/videospeed#1143]
On some websites, the upper left corner is blocked by some prompts, so the playback speed cannot be seen. I hope to set a default specific position.

**Comment by Edinho367** [comment]
I second this!! 

Everytime i try to clean up 'google photos' this extension gets exactly on the 'check' button, and this is very annoying. I'd like to move it to the bottom left corner to get rid of it without disabling it.

**Comment by jwildman16** [comment]
It would also be good just to have the controller live on top of other interface buttons.

**Comment by mi1k5h4k3pi224** [comment]
Maybe the easiest solution would be to have two input fields for top-margin and left-margin in the advanced settings, so that users can simply enter some pixel values to change the defaults.

**Comment by igrigorik** [comment]
Closing as duplicate of #716, which tracks the controller position feature request. Additional context from this issue has been added there.
